ui/app/components/app/token-cell/token-cell.js

```diff
@@ -0,0 +1,92 @@
+import classnames from 'classnames'
+import PropTypes from 'prop-types'
+import React from 'react'
+import { conversionUtil, multiplyCurrencies } from '../../../helpers/utils/conversion-util'
+import Tooltip from '../../ui/tooltip-v2'
+import AssetListItem from '../asset-list-item'
+import { useSelector } from 'react-redux'
+import { getTokenExchangeRates, getConversionRate, getCurrentCurrency, getSelectedAddress } from '../../../selectors'
+import { useI18nContext } from '../../../hooks/useI18nContext'
+import InfoIcon from '../../ui/icon/info-icon.component'
+import { formatCurrency } from '../../../helpers/utils/confirm-tx.util'
+
+export default function TokenCell ({ address, outdatedBalance, symbol, string, image, onClick }) {
+  const contractExchangeRates = useSelector(getTokenExchangeRates)
+  const conversionRate = useSelector(getConversionRate)
+  const currentCurrency = useSelector(getCurrentCurrency)
+  const userAddress = useSelector(getSelectedAddress)
+  const t = useI18nContext()
+
+  let currentTokenToFiatRate
+  let currentTokenInFiat
+  let formattedFiat = ''
+
+
+  if (contractExchangeRates[address]) {
+    currentTokenToFiatRate = multiplyCurrencies(
+      contractExchangeRates[address],
+      conversionRate
+    )
+    currentTokenInFiat = conversionUtil(string, {
+      fromNumericBase: 'dec',
+      fromCurrency: symbol,
+      toCurrency: currentCurrency.toUpperCase(),
+      numberOfDecimals: 2,
+      conversionRate: currentTokenToFiatRate,
+    })
+    formattedFiat = `${formatCurrency(currentTokenInFiat, currentCurrency)} ${currentCurrency.toUpperCase()}`
+  }
+
+  const showFiat = Boolean(currentTokenInFiat) && currentCurrency.toUpperCase() !== symbol
+
+  const warning = outdatedBalance
+    ? (
+      <Tooltip
+        interactive
+        position="bottom"
+        html={(
+          <div className="token-cell__outdated-tooltip">
+            { t('troubleTokenBalances') }
+            <a
+              href={`https://ethplorer.io/address/${userAddress}`}
+              rel="noopener noreferrer"
+              target="_blank"
+              style={{ color: '#F7861C' }}
+            >
+              { t('here') }
+            </a>
+          </div>
+        )}
+      >
+        <InfoIcon severity="warning" />
+      </Tooltip>
+    )
+    : null
+
+  return (
+    <AssetListItem
+      className={classnames('token-cell', { 'token-cell--outdated': outdatedBalance })}
+      iconClassName="token-cell__icon"
+      onClick={onClick.bind(null, address)}
+      tokenAddress={address}
+      tokenImage={image}
+      warning={warning}
+      primary={`${string || 0} ${symbol}`}
+      secondary={showFiat && formattedFiat}
```

Comment:
Nit: the `secondary` prop expects a string, but this would input a boolean (`false`) in the case where `showFiat` is `false`.

```suggestion
      secondary={showFiat ? formattedFiat : undefined}
```

Reply:
Good call.